per/factories.py

```diff
@@ -105,12 +107,22 @@ class Meta:
         model = FormPrioritization
 
 
+class AppealFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Appeal
+
+    country = factory.SubFactory(CountryFactory)
```

Comment:
Let's not use `factory.SubFactory`, this will create objects without developer knowledge which can have weird confusing related to data generation, and it's effect.